src/main/resources/openapi/audit.yaml

```diff
@@ -0,0 +1,84 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /audit/search:
+    get:
+      tags:
+        - Audit
+      summary: User can search through audit logs using various filters.
+      operationId: search
+      description: Searches through the audit log based on the filters provided.
+      parameters:
+        - in: query
+          name: startDate
+          schema:
+            format: date-time
```

Comment:
Looks like this is missing `type` - should be similar to endDate query param?
```
            type: string
            format: date-time
```

Reply:
ACCEPT